itou/templates/siae_evaluations/includes/list_item.html

```diff
@@ -1,38 +1,38 @@
-
 <div class="card my-3">
     {# navigation : anchor to scroll to a specific card when returning to this page #}
     <a class="scrolldown-anchor" id="{{ item.pk }}"></a>
     <div class="card-header">
-        {% include "siae_evaluations/includes/job_seeker_infos_for_siae.html" with job_seeker=item.job_application.job_seeker approval=item.job_application.approval state=item.compute_state reviewed_at=item.evaluated_siae.reviewed_at %}
+        {% include "siae_evaluations/includes/job_seeker_infos_for_siae.html" with job_seeker=item.job_application.job_seeker approval=item.job_application.approval state=item.compute_state_for_siae reviewed_at=item.evaluated_siae.reviewed_at %}
     </div>
 
     <div class="card-body">
 
         {% for evaluated_administrative_criteria in item.evaluated_administrative_criteria.all %}
-            <div class="row mt-1">
-                <div class="col-md-12">
-                    {% if item.evaluated_siae.reviewed_at %}
-                        {% include "siae_evaluations/includes/criterion_infos.html" with criteria=evaluated_administrative_criteria.administrative_criteria review_state=evaluated_administrative_criteria.review_state %}
-                    {% else %}
-                        {% include "siae_evaluations/includes/criterion_infos.html" with criteria=evaluated_administrative_criteria.administrative_criteria %}
-                    {% endif %}
-                </div>
-                <div class="col-md-12 mt-3">
-                    {% if evaluated_administrative_criteria.can_upload %}
-                        <a href="{% url 'siae_evaluations_views:siae_upload_doc' evaluated_administrative_criteria.pk %}"
-                           class="btn {% if evaluated_administrative_criteria.proof_url and evaluated_administrative_criteria.review_state == 'PENDING' %}btn-outline-primary {% else %}btn-primary {% endif %}float-right">
-                            {% if evaluated_administrative_criteria.proof_url %}
-                                Modifier le justificatif
-                            {% else %}
-                                Ajouter un justificatif
-                            {% endif %}
-                        </a>
-                    {% elif evaluated_administrative_criteria.proof_url %}
-                        <a href="{{ evaluated_administrative_criteria.proof_url }}" target="_blank" class="btn btn-outline-primary float-right">Visualiser le justificatif soumis</a>
-                    {% endif %}
+            {% with criteria_review_state=evaluated_administrative_criteria.review_state_for_siae %}
```

Comment:
Comme `review_state_for_siae` va appeler `hide_state_from_siae` qui va potentiellement passer en revue tous les critères on est sur un assez moche N² :cry: (vu le nombre de critères ce n'est pas grave, je sais :crying_cat_face: )

Reply:
La bonne nouvelle, c’est que `evaluated_administrative_critera.review_stae_for_siae` ne sert à rien et disparaît. :slightly_smiling_face:

Reply:
Bon, finalement `review_state_for_siae` reste.
Mis à part une cached property, je pense qu’on n’a pas trop le choix que d’itérer à nouveau.